content/en/docs/release-notes/rn-armory-operator/armory-operator-v1-2-4.md

```diff
@@ -0,0 +1,18 @@
+---
+title: v1.2.4 Armory Operator
```

Comment:
Are these versions not in lockstep w/ OSS?

Reply:
They were, we introduced a specific feature for Armory flavor when we introduce this https://github.com/armory-io/armory-operator/pull/87